src/coreclr/jit/importercalls.cpp

```diff
@@ -8332,10 +8332,7 @@ NamedIntrinsic Compiler::lookupNamedIntrinsic(CORINFO_METHOD_HANDLE method)
                     CORINFO_SIG_INFO sig;
                     info.compCompHnd->getMethodSig(method, &sig);
 
-                    int sizeOfVectorT = getSIMDVectorRegisterByteLength();
```

Comment:
This is what gave the "false dependency" on all intrinsic APIs under `System.Numerics`.

It gave a query that was `compExactlyDependsOn(InstructionSet_AVX2)` which in turn meant it opted out of any pre-jit support.

The check was instead moved down to APIs on `Vector<T>` itself -or- APIs that took/returned `Vector<T>` elsewhere.

It does not include all APIs on `System.Numerics.Vector` as we have a couple extension methods for `Vector2/3/4`, `Quaternion`, and `Plane` that live on the same class